app/views/users/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class='container'>
+<%= form_with model: @user, method: :patch do |form| %>
+  <%= form.label :username, "Username:" %>
+  <br />
+  <%= form.text_field :username, placeholder: "Enter Name"%>
+  <br /><br />
+  <%= form.label :email, "Email:" %>
+  <br />
+  <%= form.text_field :email, placeholder: "Enter Email"%>
+  <br /><br />
+  <%= form.label :password, "Password:" %>
+  <br />
+  <%= form.text_field :password, placeholder: "Enter Password"%>
+  <br /><br />
+  <%= form.submit "Edit", class: "submit" %>
+<% end %>
+</div>
+
+<style>
+  body{
+    display: grid;
+    place-items: center;
+    height: 100vh;
+  }
```

Comment:
- [ ] Never add CSS directly in the body of a view, you should add the code in the proper location, `app/assets/stylesheets`, from where the CSS pipeline compiles the styles file. 
  - Remember that you should keep your code organized and practice separation of concerns, the HTML files should have only markup, the CSS files only styles.